livre/Lib/Configuration/VolumeRendererParameters.cpp

```diff
@@ -29,6 +29,7 @@ const std::string TEXTURECACHEMEM_PARAM = "texture-cache-mem";
 const std::string TEXTUREDATACACHEMEM_PARAM = "texturedata-cache-mem";
 const std::string MINLOD_PARAM = "min-lod";
 const std::string MAXLOD_PARAM = "max-lod";
+const std::string SAMPLESPERRAY_PARAM = "samples-per-ray";
```

Comment:
I disagree here, samples is really not clear enough. ;-) This is an advanced option anyway, not something you are likely to use everyday